pkg/common/config/types.go

```diff
@@ -71,18 +71,12 @@ type Config struct {
 		Zone   string `gcfg:"zone"`
 		Region string `gcfg:"region"`
 	}
-	// FeatureStatesConfig contains feature states configmap info specific to supervisor cluster
-	FeatureStatesConfig FeatureStatesConfigInfo
-
-	// InternalFeatureStatesConfig contains feature states configmap info specific to pvCSI and vanilla drivers
-	// NOTE: Do not edit this. Only to be used for dev and testing purposes.
-	InternalFeatureStatesConfig FeatureStatesConfigInfo
 }
 
-// FeatureStatesConfigInfo is the details about feature states configmap
+// FeatureStatesConfigInfo contains the details about feature states configmap
 type FeatureStatesConfigInfo struct {
```

Comment:
FeatureStatesConfigInfo is no longer part of vsphere.conf file. should we move this out of `pkg/common/config/types.go`?

Reply:
I wanted it to be in a `types.go` file common to csi-driver and syncer, therefore I let it be here. Would you prefer to keep it inside `pkg/csi/service/common/types.go` instead?